src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const perDay = 40;
+  let total = days * perDay;
```

Comment:
Use 'const' instead of 'let' for 'total' since it's not being reassigned but only modified, which can still be done with a constant.